Makefile

```diff
@@ -769,6 +772,8 @@ dist/wasm/Makefile.basic: MERKLE_BUNDLE = -bundle 'MerkleTree,MerkleTree.*'
 dist/wasm/Makefile.basic: CTR_BUNDLE =
 dist/wasm/Makefile.basic: DEFAULT_FLAGS += -bundle 'EverCrypt,EverCrypt.*'
 
+dist/wasm/Makefile.basic: BLAKE2_BUNDLE += -bundle 'Hacl.Blake2b_256,Hacl.Blake2s_128'
```

Comment:
bad merge: I believe I had already added the right thing for you just above, so this line is superfluous